tests/test_cxl_component_virtual_switch_manager.py

```diff
@@ -40,7 +40,13 @@ def test_virtual_switch_manager_init():
         switch_connection_manager=switch_connection_manager, port_configs=port_configs
     )
     switch_configs = [
-        VirtualSwitchConfig(upstream_port_index=0, vppb_counts=3, initial_bounds=[1, 2, 3])
+        VirtualSwitchConfig(
+            upstream_port_index=0,
+            vppb_counts=3,
+            initial_bounds=[1, 2, 3],
+            irq_host="127.0.0.1",
+            irq_port=8500,
```

Comment:
Because `make test` runs test cases in parallel in random order, each test case must be _guaranteed_ to have different port numbers. If these test cases all use 8500, when run in-parallel, some of them would fail (and this failure would occur randomly). Please take a look at the usage of code like:
```
    port = BASE_TEST_PORT + pytest.PORT.TEST_1
```